game/outfits.rpy

```diff
@@ -0,0 +1,304 @@
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_MORNING_ALT if _use_alt_outfit else DEFAULT_OUTFIT_MORNING,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_MORNING_ALT if _use_alt_outfit else DEFAULT_OUTFIT_MORNING,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_CASUAL_WEEKEND_ALT if _use_alt_outfit else DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_CASUAL_WEEKEND_ALT if _use_alt_outfit else DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_NIGHT:DEFAULT_OUTFIT_NIGHT
+    }
+
+    DEFAULT_OUTFIT_SCHEDULE_WEEKDAY_MEDIUM_AFFINITY = {
+        utils.TIME_BLOCK_EARLY_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_CASUAL_WEEKDAY,
+        utils.TIME_BLOCK_NIGHT: DEFAULT_OUTFIT_CASUAL_WEEKDAY
+    }
+
+    DEFAULT_OUTFIT_SCHEDULE_WEEKEND_MEDIUM_AFFINITY = {
+        utils.TIME_BLOCK_EARLY_MORNING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_NIGHT: DEFAULT_OUTFIT_CASUAL_WEEKEND
+    }
+
+    DEFAULT_OUTFIT_SCHEDULE_WEEKDAY_LOW_AFFINITY = {
+        utils.TIME_BLOCK_EARLY_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_UNIFORM,
+        utils.TIME_BLOCK_NIGHT: DEFAULT_OUTFIT_UNIFORM
+    }
+
+    DEFAULT_OUTFIT_SCHEDULE_WEEKEND_LOW_AFFINITY = {
+        utils.TIME_BLOCK_EARLY_MORNING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_MID_MORNING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_LATE_MORNING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_AFTERNOON: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_EVENING: DEFAULT_OUTFIT_CASUAL_WEEKEND,
+        utils.TIME_BLOCK_NIGHT: DEFAULT_OUTFIT_CASUAL_WEEKEND
+    }
+
+    def set_outfit(outfit):
+        """
+        Sets Natsuki's appearance using the given outfit.
+        """
+        global current_outfit_name
+        
+        if not isinstance(outfit, Outfit):
+            raise Exception("Outfit given is not an Outfit-class object")
+            return
+
+        if outfit.clothes is None:
+            raise Exception("Outfit clothing cannot be None")
+            return
+
+        if outfit.hairstyle is None:
+            raise Exception("Outfit hairstyle cannot be None")
+            return
```

Comment:
This should be managed by the `Outfit` constructor. We shouldn't be validating this later